docs/Contributing/API-for-contributors.md

```diff
@@ -2276,6 +2276,9 @@ Returns the host information about the device that makes the request.
   "license": {
     "tier": "free",
     "expiration": "2031-01-01T00:00:00Z"
+  },
+  "global_config": {
+    "mdm_enabled_and_configured": false
```

Comment:
What do you think of going with: 

```json
{
  "global_config": {
    "mdm": {
      "enabled_and_configured": false
    }
  }
}
```

That way we keep the same structure and naming convention.

Reply:
the response is actually what are you suggesting in both places, it's just that the documentation is wrong 😅 lol

Reply:
great catch, thank you! I have updated the docs to match the API in both places.